quarkus-distroless-image.yaml

```diff
@@ -0,0 +1,17 @@
+- name: debian
+  version: stable-slim
```

Comment:
Why debian? Didn't you example use UBI?

Reply:
distroless images are built from debian, I think it's cleaner if the lib we're adding comes from the same distribution

Reply:
ah ok!